modules/renter/consts.go

```diff
@@ -34,4 +34,12 @@ var (
 		Standard: time.Minute * 15,
 		Testing:  10 * time.Second,
 	}).(time.Duration)
+
+	// maxChunkCacheSize determines the maximum number of chunks that will be
+	// cached in memory.
+	maxChunkCacheSize = build.Select(build.Var{
```

Comment:
is this a replacement of the download pieces thing? The combination of download pieces and upload pieces?